Source/Example Measurements/NI-Digital SPI/NIDigitalSPI.sfp

```diff
@@ -0,0 +1,35 @@
+	</SourceModelFeatureSet>
+	<UnifiedTask Id="61783f6f83fb4883980167b7e39c8572" LayoutInformation="{&quot;Format Version&quot;:1,&quot;Application Format Version&quot;:23.0,&quot;Layout Information&quot;:{&quot;Sections&quot;:[{&quot;Columns Progressive Count&quot;:1,&quot;Layout Information Presentation&quot;:&quot;ConfigurationWithVisualization&quot;,&quot;Columns&quot;:[{&quot;Containers Progressive Count&quot;:1,&quot;Layout Information Index&quot;:0,&quot;Containers&quot;:[{&quot;Container Model Defition&quot;:&quot;{http://www.ni.com/InstrumentFramework/ScreenDocument}Screen&quot;,&quot;Container Assignment State&quot;:&quot;Available&quot;,&quot;Container Identifier&quot;:&quot;{http://www.ni.com/InstrumentFramework/ScreenDocument}Screen.task&quot;,&quot;Container Instrument Name&quot;:&quot;NI-Digital SPI (Py)&quot;,&quot;Layout Information Name&quot;:&quot;NI-Digital SPI (Py) 1&quot;,&quot;Layout Information Index&quot;:0,&quot;Layout Information Presentation&quot;:&quot;ConfigurationWithVisualization&quot;,&quot;Items&quot;:[{&quot;Identification Number&quot;:4294963201,&quot;Model Definition Type&quot;:&quot;{http://www.ni.com/InstrumentFramework/ScreenDocument}Screen&quot;,&quot;Parent Or Group Name&quot;:&quot;Measurements&quot;,&quot;Panel Type&quot;:&quot;NI-Digital SPI (Py)&quot;,&quot;Layout Information Name&quot;:&quot;ni.examples.NIDigitalSPI_Python&quot;,&quot;Layout Information Index&quot;:0}]}]}]},{&quot;Columns Progressive Count&quot;:0,&quot;Layout Information Presentation&quot;:&quot;ConfigurationOnly&quot;}]}}" PinAwareInformation="{&quot;Format Version&quot;:1,&quot;Application Format Version&quot;:23.0,&quot;Selected Sites&quot;:[{&quot;Site Number&quot;:0}]}" xmlns="http://www.ni.com/InstrumentFramework/Document/SourceModel/UnifiedSoftFrontPanel" />
+	<EnvoyManagerFile Id="6c31f1cca31b4b2fa7969d75081ad3bd" xmlns="http://www.ni.com/PlatformFramework">
+		<EnvoyManagerRootEnvoy Id="e948a5811cee45b3a3807dae45535633" ModelDefinitionType="EnvoyManagerRootEnvoy" Name="RootEnvoy" />
+		<NameScopingEnvoy AutomaticallyResolveUp="True" Id="78b3f18a9db84d6ea5f2e3e81919b7b2" Name="NIDigitalSPI.sfp" NameTracksFileName="True" />
+		<EmbeddedDefinitionReference Id="a4b06c4b696e4fc69fa02b8af0aa05a4" ModelDefinitionType="{http://www.ni.com/InstrumentFramework/ScreenDocument}Screen" Name="{http\://www.ni.com/InstrumentFramework/ScreenDocument}Screen.task">
+			<Screen ClientId="{4e624bc4-ee53-476d-8221-e267b8671382}" ConfigurationParameters="{&quot;@type&quot;:&quot;type.googleapis.com/ni.measurementlink.measurement.v1.MeasurementConfigurations&quot;,&quot;pinNames&quot;:[&quot;SPI_PINS&quot;]}" DisplayName="NI-Digital SPI (Py)" Id="7319682d2bea49af95dbd40688eb3193" PanelPresentation="ConfigurationWithVisualization" ServiceClass="ni.examples.NIDigitalSPI_Python" xmlns="http://www.ni.com/InstrumentFramework/ScreenDocument">
```

Comment:
The measurement panel appears to be pointing at a service class (the Python version) for a different service then this example provides. The snapshot of the UI captured in the sfp is also for a .measui. This isn't a big deal as it should auto update. However, if the service isn't install/running, then this is the default view of the UI that is displayed, and it doesn't match the current UI for the service.

Reply:
I tried to load this .sfp and it didn't really load correctly. We should just delete it and recreate it.

Reply:
Fixed by recreating this SFP file as suggested. This file was stale and I was actually updating a temporary NIDigitalSPI.sfp file I had on my desktop.